grug/storage/src/key.rs

```diff
@@ -437,7 +480,7 @@ where
 /// ```
 ///
 /// is also returned.
-pub(crate) fn split_first_key(key_elems: u8, value: &[u8]) -> (Vec<u8>, &[u8]) {
+pub fn split_first_key(key_elems: u8, value: &[u8]) -> (Vec<u8>, &[u8]) {
```

Comment:
No reason to make this public?

Reply:
This is a leftover cause was implementing PK for CodesStatus under CodesStatus declaration